### PR TITLE
Sync web overlays in stage manager

### PR DESCRIPTION
**Pull request by misslivirose**
**Test Plan**
1. In a domain where you have rez rights, run https://hifi-content.s3-us-west-1.amazonaws.com/liv/Prototypes/StageManager/stageManager.js

2. Open the stage manager app and paste in a URL , then click SET

3. Spawn a web overlay using the tablet controls and confirm that the overlay goes to your specified URL

4. Remove the web overlay and confirm that the overlay goes away

Should test for multiple people in the domain. The overlay spawns around {x: 98.5507, y: 6.56, z: 23.7452}

### CODE REVIEW

**Comment by misslivirose** [comment]
I need to remove the server script, it's no longer used

File: DomainContent/FaceToFace/StageManager/webOverlaySpawner.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        overlayProperties.url = _url; 
+        overlay = Overlays.addOverlay('web3d', overlayProperties);
+    }
+
+    var WebOverlaySpawner = function(){};
+
+    WebOverlaySpawner.prototype = {
+        preload: function(entityID) {
+            _entityID = entityID;
+            _url = JSON.parse(Entities.getEntityProperties(entityID, 'userData').userData).url;
```
Comment (MiladNazeri) [inline DomainContent/FaceToFace/StageManager/webOverlaySpawner.js]: maybe try / catch check unless you are certain you will always get userData

File: DomainContent/FaceToFace/StageManager/stageManager.js [hunk 2]
```diff
@@ -63,8 +65,26 @@
...
             event = JSON.parse(event);
             switch (event.type) {
+                case 'setWebOverlayUrl' : 
+                    webOverlayURL = event.url;
+                    break;
+                case 'spawnWebOverlay' :
+                    webOverlaySpawner = Entities.addEntity({
+                        type: 'Box',
+                        position: stageCenterPosition,
+                        visible: true,
```
Comment (MiladNazeri) [inline DomainContent/FaceToFace/StageManager/stageManager.js]: Is this meant to be visible: true?

Comment (MiladNazeri) [reply DomainContent/FaceToFace/StageManager/stageManager.js]: Just tried spawning it and you can see it from the stage.  Wasn't sure if it was intended

**Pull request merged**